7_fetalSC_vs_LGR5+_and_LGR5-/0_Haber_2017_create_anndata_file.ipynb

```diff
@@ -0,0 +1,2898 @@
+{
```

Comment:
<p>Let's rename the categories so they don't contain spaces in ebtween. Please use _ instead of spaces. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>